crates/nuon/src/to.rs

```diff
@@ -8,39 +8,54 @@ use nu_protocol::{Range, ShellError, Span, Value};
 
 use std::ops::Bound;
 
+/// control the way Nushell [`Value`] is converted to NUON data
+pub enum ToStyle {
+    /// no indentation at all
+    ///
+    /// `{ a: 1, b: 2 }` will be converted to `{: 1, b: 2}`
```

Comment:
and `a` is missing here in the "converted to" example.

Reply:
ooh yeah, good catch !!
should be fixed in https://github.com/nushell/nushell/pull/12608